test/test-remote/looker/src/metrics/dummyseries.ts

```diff
@@ -205,7 +205,7 @@ export class MetricSeries extends TimeseriesBase {
 
   protected buildLabelSetFromOpts(opts: MetricSeriesOpts): LabelSet {
     // Merge the metric name into it using the well-known special prom label
-    // __name__. Always set `uniquename` and `__name__`. If `opts.labelset` is
```

Comment:
why did we remove "`uniquename` and`" here? I think the code indeed always sets `uniquename=<uniquename>` :)

Reply:
I think this had been a change that I had started and then backed out... I'll revert the comment change